client-report/webpack.config.js

```diff
@@ -17,7 +17,7 @@ module.exports = {
     new webpack.optimize.OccurenceOrderPlugin(),
     new webpack.DefinePlugin({
       "process.env": {
```

Comment:
Should this be `"process.env.NODE_ENV": JSON.stringify("production")` to avoid clobbering other stuff in `process.env` (as described in the `DefinePlugin` docs: https://webpack.js.org/plugins/define-plugin/)?

Reply:
I figure the 'stringify' is for variables that are not guaranteed to be strings, since it's not really necessary to stringify a string. But we can be consistent and keep it here too.

Reply:
I think I got excited to merge this yesterday and missed this; The problem isn't stringify, it's that we should be setting `process.env.NODE_ENV` rather than `process.env`, since the latter could overwrite other settings.